docs/platform-install.md

```diff
@@ -14,6 +14,8 @@ We currently only have installation instructions for MacOS. If you're using Wind
 xcode-select --install
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 brew install rbenv ruby-build node
+gem install liquid-c -v '4.0.0' --source 'https://rubygems.org/' -- --with-cflags=\"-Wno-compound-token-split-by-macro\"
```

Comment:
This line is not required for all installs. It should be in an FAQ section in this doc for if you get an error when installing `liquid-c`